app.py

```diff
@@ -12,6 +12,8 @@
 app.config.from_pyfile('settings.py',silent=True)
 if "VAULT_ADDR" in os.environ:
     app.config['VAULT_URL'] = os.environ['VAULT_ADDR']
+if "VAULT_SKIP_VERIFY" in os.environ:
```

Comment:
if the value is in the environment you set it to True?  So below in the settings.py where you do VAULT_SKIP_VERIFY = False... would actually set this to true